cds/modules/theme/static/js/cds-build.js

```diff
@@ -30,7 +30,7 @@
       comments: false
     },
     compress: {
-      drop_console: false,
+      drop_console: true,
```

Comment:
We are not using ``requirejs`` anymore :/

Reply:
shall we delete the file then?